test_utils/fake_catalog_api.py

```diff
@@ -481,8 +481,10 @@ def get_catalog_courses(catalog_id):
     Arguments:
         catalog_id (int): Catalog ID
 
-    Returns:
+    Returns
+    -------
```

Comment:
@snowcrshd Please run `make docs` and make sure these underlining does not break documentation layout and does not introduce "Return" records into navigation menu (left sidebar).